src/lib/features/release-plans/release-plan-milestone.ts

```diff
@@ -5,7 +5,7 @@ export interface ReleasePlanMilestone {
     name: string;
     sortOrder: number;
     releasePlanDefinitionId: string;
-    startedAt?: Date;
+    startedAt?: string;
```

Comment:
this is what enterprise repo expects and since it depends on openapi schema which is string. since we don't need date manipulations in the code we can return string and pass it up to the API layer